osu.Game/Graphics/Cursor/OsuContextMenuContainer.cs

```diff
@@ -13,11 +13,18 @@ public partial class OsuContextMenuContainer : ContextMenuContainer
         [Cached]
         private OsuContextMenuSamples samples = new OsuContextMenuSamples();
 
-        public OsuContextMenuContainer()
+        private OsuContextMenu menu = null!;
+
+        protected override Menu CreateMenu() => menu = new OsuContextMenu(true);
+
+        public void ShowMenu()
```

Comment:
I'm not sure it's meaningful for this to exist, because the menu requires a target to pull its items from.

Reply:
oh yea, only the close menu is meaningful cuz there no need the target, the show menu needs a parent to perform the action

Reply:
The method was removed at the latest commit.